### ISSUE TITLE
Enhanced dependencies in en_ewt-ud-train.conllu

### ISSUE THREAD

**Issue by neutralino** [Oneplus/twpipe#3]
Hi, apologies if this is already documented somewhere.  I'm trying to include the training set `en_ewt-ud-train.conllu` from https://github.com/UniversalDependencies/UD_English-EWT, but run into an error:
```
libc++abi.dylib: terminating with uncaught exception of type boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::bad_lexical_cast> >: bad lexical cast: source type value could not be interpreted as target
```
due to the following line in that training set
```
8.1 reported report VERB VBN Tense=Past|VerbForm=Part|Voice=Pass _ _ 5:conj:and CopyOf=5
```
It looks like it's due to the underscore in the HEAD field.  Is the pipeline expected to be compatible with null nodes / enhanced dependencies?  Thanks.

**Comment by Oneplus** [comment]
I've fixed it in 6d934d248f898176e80e1d0ed19ad0a63a904c38, please update

**Comment by neutralino** [comment]
Great, thank you.
